vsts/pipelines/_signbinary.yml

```diff
@@ -7,7 +7,7 @@ steps:
 - powershell: |
    Write-Host "##vso[task.setvariable variable=SignType]real"
   displayName: 'Set sign binary variable'
-  condition: and(in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'))
+  condition: and(in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'), or(startsWith(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'],'refs/heads/patch/' )))
```

Comment:
Loooooooooong line 😄 ...Could you see if you can break it within the 120 limit? https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines

Reply:
done !